src/activities/GianaBhateja/GianaBhateja.qml

```diff
@@ -0,0 +1,575 @@
+/* GCompris - GianaBhateja.qml
+ *
+ * Copyright (C) 2016 YOUR NAME <xx@yy.org>
+ *
+ * Authors:
+ *   <THE GTK VERSION AUTHOR> (GTK+ version)
+ *   YOUR NAME <YOUR EMAIL> (Qt Quick port)
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.1
+import GCompris 1.0
+import QtQuick.Controls 1.0
+
+import "../../core"
+import "GianaBhateja.js" as Activity
+
+ActivityBase {
+    id: activity
+
+    property color buttonColor: "lightblue"
+    property color onHoverColor: "gold"
+    property color borderColor: "white"
+
+    onStart: focus = true
+    onStop: {}
+
+
+   /* Image {
+        id: icon
+        //source: "GianaBhateja/arrow1.png"
+        source: "qrc:/gcompris/src/activities/GianaBhateja/resource/img2.svg"
+        anchors.horizontalCenter: parent.horizontalCenter
+    }*/
+    Text {
```

Comment:
We have a GCText item to handle texts. It is used the same as Text but has common properties for all the texts (same font, letter spacing...).